### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by bbrown949**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,10 +18,46 @@
...
+            <div><label for="title">Title</label></div>
+            <input name="text" type="text" id="title" required/>
+            <div><label for="url">Photo URL</label></div>
+            <input type="url" name="url" id="url" class="url" pattern="https://.*" size="30" required>
+          </div>
+          <div class="column-full">
+            <div>
+              <label for="notes">Notes</label>
+            </div>
+            <textarea name="comment" id="comments" cols="80" rows="8">
```
Comment (shawnkost) [inline index.html]: This `id` should be `notes`, so that it matches the `label` `for` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Bryan!

File: index.html (hunk 1 above)
Comment (bbrown949) [reply index.html]: updated

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**